src/mmw/js/src/analyze/views.js

```diff
@@ -638,6 +640,60 @@ var ClimateTableView = Marionette.CompositeView.extend({
     }
 });
 
+var StreamTableRowView = Marionette.ItemView.extend({
+    tagName: 'tr',
+    template: streamTableRowTmpl,
+
+    templateHelpers: function() {
+        var order = this.model.get('order'),
+            displayOrder = (function() {
+                switch (order) {
+                    case 1:
+                        return order.toString() + 'st';
+                    case 2:
+                        return order.toString() + 'nd';
+                    case 3:
+                        return order.toString() + 'rd';
+                    case 999:
+                        return 'Other';
+                    default:
+                        return order.toString() + 'th';
+                }
+            })();
+
+        return {
+            displayOrder: displayOrder,
+            lengthkm: this.model.get('lengthkm'),
+            slopepct: this.model.get('slopepct')
+        };
+    }
+});
+
+var StreamTableView = Marionette.CompositeView.extend({
+    childView: StreamTableRowView,
+    childViewContainer: 'tbody',
+    template: streamTableTmpl,
+
+    onAttach: function() {
+        $('[data-toggle="table"]').bootstrapTable();
+    },
+
+    templateHelpers: function() {
+        var data = this.collection.toJSON(),
```

Comment:
Since `lodash(data)` is used numerous times below, consider doing that once here so it is reused.